v3.9/rockcraft.yaml

```diff
@@ -0,0 +1,40 @@
+name: multus
+summary: Rock for the Multus thin CNI.
+description: >
+  This rock is a drop in replacement for the
+  ghcr.io/k8snetworkplumbingwg/multus-cni thin image.
+version: v3.8
+license: Apache-2.0
+
+base: bare
+build-base: ubuntu@22.04
+platforms:
+  amd64:
+  arm64:
+
+services:
+  multus:
+    command: /entrypoint.sh
+    override: replace
+    startup: enabled
+
+parts:
+  build-deps:
+    plugin: nil
+    build-snaps:
+      - go/1.17/stable
```

Comment:
that Go version looks quite outdated, any reason for using it?

Reply:
The reference image used go 1.17 but I'll try bumping the version.

https://github.com/k8snetworkplumbingwg/multus-cni/blob/ca8c9c579100648b88774cb9f79223eb213373a2/images/Dockerfile#L2